src/components/EditProfile.vue

```diff
@@ -48,7 +49,7 @@
 var _ = require('lodash')
 import PostModal from './PostModal.vue'
 import axios from 'axios'
-import Crypto from 'crypto-js'
+// import Crypto from 'crypto-js'
```

Comment:
Can we just remove this? Commented out code can get messy at times (Unless you think we will need it later).